_docs/2017-01-01-index.md

```diff
@@ -78,9 +78,8 @@ process very time efficient.
 #### Security
 
 Your money is as secure as the _secret key_ that allows spending it.
-That means that storing your secret key in a safe on a laptop
-disconnected from the Internet is equivalent to having banknotes in a
-password-protected lockbox in a safe. Absolutely nobody can steal this
+That means that storing your secret key on a USB flash drive in a safe
```

Comment:
This is absolutely minor, but try to avoid repeating pronouns when they clash with your conjunction.
"That means that storing your secret key" -> "This means that storing your secret key"